### PR TITLE
fix(front): Fix wording issues

### PR DESCRIPTION
**Pull request by fatonramadani**

### CODE REVIEW

File: frontend/src/lib/components/flows/content/FlowRetries.svelte [hunk 1]
```diff
@@ -71,14 +71,14 @@
...
-		<span class="text-xs font-bold">Seconds</span>
+		<span class="text-xs font-bold">Initial delay</span>
 		<input bind:value={$flowStore.value.retry.exponential.seconds} type="number" />
 	{:else}
 		<span class="text-xs font-bold">Attempts</span>
 		<input type="number" disabled />
 		<span class="text-xs font-bold">Mulitplier</span>
 		<input type="number" disabled />
-		<span class="text-xs font-bold">Seconds</span>
+		<span class="text-xs font-bold">Initial delay</span>
```
Comment (fatonramadani) [inline frontend/src/lib/components/flows/content/FlowRetries.svelte]: @lplit Did I understand your suggestion correctly ?

Comment (lplit) [reply frontend/src/lib/components/flows/content/FlowRetries.svelte]: Yes, that's exactly what I had in mind!

**Review by lplit — APPROVED** [review]

File: frontend/src/lib/components/flows/content/FlowRetries.svelte (hunk 1 above)
Comment (rubenfiszel) [inline frontend/src/lib/components/flows/content/FlowRetries.svelte]:
```suggestion
		<span class="text-xs font-bold">Initial delay (seconds)</span>
```

Comment (fatonramadani) [reply frontend/src/lib/components/flows/content/FlowRetries.svelte]: I'll add that everywhere.

**Review by rubenfiszel — APPROVED** [review]

**Pull request merged**